latest/index.bs

```diff
@@ -230,37 +228,944 @@ Metadata {#metadata}
+]
+```
+</div>
+
+#### sequence
+
+A `sequence` transformation consists of an ordered array of coordinate transformations, and is invertible if and only if every
+coordinate transform in the array is invertible.
+
+<dl>
+  <dt><strong>transformations</strong></dt>
+  <dd>A non-empty array of transformations.</dd>
+</dl>
+
+The `input` and `output` fields SHOULD be omitted for transformations that are in the list of transformations of a `sequence`.
+
+<div class=example>
+
+This sequence:
+
+<pre class=include-code>
+path: examples/transformations/sequence.json
+highlight: json
+</pre>
+
+describes the function
+
+```
+x = (i + 0.1) * 2
+y = (j + 0.9) * 3
+```
+
+and is invertible.
+</div>
+
+
+#### coordinates and displacements
+
+`coordinates` and `displacements` transformations store coordinates or displacements in an array and interpret them as a transformation.
+Applying the transformation amounts to looking up the appropriate locations in the array and interpolating.
+`coordinates` and `displacements` transformations are not invertible in general, but implementations
+MAY approximate their inverses. Metadata for these coordinate transforms have the following field: 
+
+<dl>
+  <dt><strong>path</strong></dt>
+  <dd>  The location of the coordinate array in this (or another) containter.</dd>
+  <dt><strong>url</strong></dt>
+  <dd>  An optional URL to the container in which the coordinate field array is stored. If not provided,
+        the provided `path` MUST exist in this container.</dd>
+  <dt><strong>interpolation</strong></dt>
+  <dd>  The `interpolation` attributes MAY be provided. It's value indicates
+        the interpolation to use if transforing points not on the array's discrete grid.
+        Values could be:
+        <ul>
+            <li><code>linear</code> (default)</li>
+            <li><code>nearest</code></li>
+            <li><code>cubic</code></li>
+        </ul></dd>
+</dl>
+
+The array data at `path` MUST define space and coordinate transform metadata:
+
+* space metadata MUST have exactly one axis with `"type" : "coordinate"`
+* Every axis name in the `coordinateTransform`'s `input` MUST appear in the space
+* The array dimension corresponding to the coordinate axis MUST have length greater than or equal to the dimensionality of the `coordinateTransform` `output`
+* SHOULD have `name` equal to the `name` of the corresponding `coordinateTransform`.
+
+For `coordinates`:
+
+* If a `coordinates`' input space has dimensionality `N`, then the array data at `path` MUST have dimensionality equal to `(ni + 1)`.
+* space metadata MUST have exactly one axis with `"type" : "coordinate"`
+
+For `displacements`:
+* space metadata MUST have exactly one axis with `"type" : "displacement"`
+* `input` and `output` MUST have an equal number of dimensions.
+
+For example, in 1D:
+```
+{
+    "name" : "a coordinate field transform",
+    "type": "coordinates",
+    "path" : "i2xCoordinates",
+    "input" : ["i"],
+    "output" : ["x"],
```

Comment:
Are coordinate and displacement transformations meant to be like `"byDimensions`" transformations?

AFAICT input and output dimensions for these transforms would fall under the general rule, and must be strings referring to coordinate spaces. I think this needs further specification.

Reply:
nope, that's just an old example I haven't fixed yet.  nice catch